data_steward/validation/main.py

```diff
@@ -107,67 +107,83 @@ def _upload_achilles_files(hpo_id):
 
 @api_util.auth_required_cron
 def validate_hpo_files(hpo_id):
-    logging.info(' Validating hpo_id %s' % hpo_id)
-    bucket = gcs_utils.get_hpo_bucket(hpo_id)
-    bucket_items = gcs_utils.list_bucket(bucket)
-
-    # separate cdm from the unknown (unexpected) files
-    found_cdm_files = []
-    unknown_files = []
-    for bucket_item in bucket_items:
-        if _is_cdm_file(bucket_item):
-            found_cdm_files.append(bucket_item)
-        else:
-            if bucket_item['name'].lower() in common.IGNORE_LIST + common.CDM_FILES:
+    hpo_ids = [hpo_id]
```

Comment:
Having several changed lines in a critical section like this makes me a bit nervous. Perhaps we can avoid it. What do you think about creating a separate endpoint/function which just invokes the intact function for each `hpo_id`?

Reply:
there's flask app +  cron problem. The way it's structured now, we can only call the endpoint and not a function and we can't call the endpoint because that can only be done by cron.

Reply:
Ack, I thought that might be the case. Thanks for the info. Could we create a dumb function just for the endpoint?

    @api_util.auth_required_cron
    def validate_hpo_files_endpoint(hpo_id):
      return validate_hpo_files(hpo_id)

Reply:
possible.